GOVERNANCE.md

```diff
@@ -194,11 +193,19 @@ Voting for first election is restricted to establish stable governance, and to d
 ### Core Contributors
 Core Contributors are those who have provided consistent and meaningful contributions to ArviZ.
 These can be, but are not limited to, code contributions, community contributions, tutorial
-development etc.
+development etc. Core Contributors will be given the ability to manage the ArviZ github
```

Comment:
This is only for core contributors?

Reply:
We have to make sure we follow whatever we decide here before merging. IIUC, the writing above would mean people with merge privileges who is not core contributor has to become core contributor or lose the merge privileges

Reply:
I agree, we need to make things consistent before we merge this.

Reply:
Yeah, I agree with Oriol's interpretation.